client/lib/products-values/constants.js

```diff
@@ -283,6 +268,23 @@ export const getJetpackProducts = () => {
 			},
 			slugs: JETPACK_SEARCH_PRODUCTS,
 		} );
+	isEnabled( 'jetpack/scan-product' ) &&
+		output.push( {
+			title: translate( 'Jetpack Scan' ),
+			description:
+				'Automatic scanning and one-click fixes keep your site one step ahead of security threats.',
```

Comment:
```suggestion
			description: getJetpackProductsDescriptions()[ PRODUCT_JETPACK_SCAN ],
```

Reply:
Yep, great catch — especially since it wasn't even translating the description.